src/App.js

```diff
@@ -2,26 +2,26 @@ import React from 'react';
 import Header from './Header.js';
 import Main from './Main.js';
 import Footer from './Footer.js';
-import data from './data.json';
+import Data from './data.json';
```

Comment:
Really good work on the lab!

- for line 5 (and those that relate to the import), we will want to change this to lowercase `data`. Uppercase is typically reserved for components (classes). 👍